ods_ci/tasks/Resources/Provisioning/GPU/AMD/amd_operator.sh

```diff
@@ -196,12 +198,13 @@ spec:
   publisher: RHOAI QE
   sourceType: grpc
 EOF
+    sleep 5s
```

Comment:
adding a sleep because I noticed the catalog source pod being created twice: first time it's deleted immediately